batch/Makefile

```diff
@@ -0,0 +1,48 @@
+PROJECT := $(shell gcloud config get-value project)
+ZONE := us-central1-a
+KUBERNETES_SERVER_URL := https://104.198.230.143
+
+BATCH_LATEST = gcr.io/$(PROJECT)/batch:latest
+BATCH_IMAGE = gcr.io/$(PROJECT)/batch:$(shell docker images -q --no-trunc batch | sed -e 's,[^:]*:,,')
+
+BATCH_WORKER_LATEST = gcr.io/$(PROJECT)/batch-worker:latest
+BATCH_WORKER_IMAGE = gcr.io/$(PROJECT)/batch-worker:$(shell docker images -q --no-trunc batch-worker | sed -e 's,[^:]*:,,')
+
+PYTHONPATH := $${PYTHONPATH:+$${PYTHONPATH}:}../hail/python:../gear:../web_common
+PYTHON := PYTHONPATH=$(PYTHONPATH) python3
+
+check:
+	$(PYTHON) -m flake8  --config ../setup.cfg batch
+	$(PYTHON) -m pylint --rcfile ../pylintrc batch --score=n
+
+build:
+	make -C ../docker build
+	-docker pull $(BATCH_LATEST)
+	python3 ../ci/jinja2_render.py '{"service_base_image":{"image":"service-base"}}' Dockerfile Dockerfile.out
+	docker build -t batch -f Dockerfile.out --cache-from batch,$(BATCH_LATEST),service-base ..
+	-docker pull python:3.6-slim-stretch
+	-docker pull $(BATCH_WORKER_LATEST)
+	docker build -t batch-worker -f Dockerfile.worker --cache-from batch-worker,$(BATCH_WORKER_LATEST),python:3.6-slim-stretch ..
+
+push: build
+	docker tag batch $(BATCH_LATEST)
+	docker push $(BATCH_LATEST)
+	docker tag batch $(BATCH_IMAGE)
+	docker push $(BATCH_IMAGE)
+	docker tag batch $(BATCH_WORKER_LATEST)
+	docker push $(BATCH_WORKER_LATEST)
+	docker tag batch $(BATCH_WORKER_IMAGE)
```

Comment:
shouldn't these be batch-worker?

Reply:
Isn't it?

 > BATCH_WORKER_IMAGE = gcr.io/$(PROJECT)/batch-worker:...

Maybe I don't understand the question.

Reply:
I thought the tags should be batch-worker.

Reply:
Oops, 100%, good catch!